### PR TITLE
Better documentation for LIDAR profile integration

### PR DESCRIPTION
**Pull request by llienher**
Fix [GSGMF-1815](https://jira.camptocamp.com/browse/GSGMF-1815)

### CODE REVIEW

**Comment by llienher** [comment]
@arnaud-morvan If you can see the CI error with jinja templating.

**Comment by arnaud-morvan** [comment]
There are cookie cutter variables in this file, we cannot ignore cookie cutter templating on it, so I've added `{% raw %} {% endraw %} around the javascript code with `{{`.

**Comment by llienher** [comment]
@arnaud-morvan Thanks for the fix !

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 1]
```diff
@@ -277,21 +277,23 @@ Changes to apply
    converted to ``true``/``false``, and the properties ``mapcache_internal`` and ``mapcache`` should be
    removed.
 
-4. For the LIDAR profile you should add/modify in your controller JavaScript file:
+4. For the LIDAR profile you should add/modify your controller constructor, in the interface JavaScript file:
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:280]: The import is not in the constructor...

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 2]
```diff
@@ -307,21 +309,60 @@ Changes to apply
...
     +   });
+    +
+    +    /**
+    +     * @type {boolean}
+    +     */
+    +    this.drawLidarprofilePanelActive = false;
+    +
+    +    // Disable the map interaction (queries) when the lidar is active
+    +    const drawLidarprofilePanelActive = new ngeoMiscToolActivate(this, 'drawLidarprofilePanelActive');
+    +    this.ngeoToolActivateMgr.registerTool('mapTools', drawLidarprofilePanelActive, false);
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:320]: This should be already present, then it's wrong...

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 3]
```diff
@@ -307,21 +309,60 @@ Changes to apply
...
     ...
     ```
 
-    In you HTML file the components should be added like this:
+    The interface HTML file needs a few changes.
+    In the body balise, change it to add the 'gmf-lidarprofile-chart-active' class with the following code:
+    ```html
+    ...
+    -  <body ng-class="{'gmf-profile-chart-active': !!profileChartActive, 'gmf-query-grid-active': !!mainCtrl.queryGridActive}" ng-keydown="mainCtrl.onKeydown($event)">
+    +  <body ng-class="{'gmf-profile-chart-active': !!profileChartActive, 'gmf-lidarprofile-chart-active': !!mainCtrl.lidarProfileFooterActive, 'gmf-query-grid-active': !!mainCtrl.queryGridActive}" ng-keydown="mainCtrl.onKeydown($event)">
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:329]: This should be already present, then it's wrong...

**Review by sbrunner — CHANGES_REQUESTED** [review]
I stop my review because the changes speak about adding the LIDAR profile but he should ash about updating the LIDAR profile

**Comment by llienher** [comment]
@sbrunner I made the doc shorter as it is of course updating, not add from nothing. :-)

**Review by sbrunner — APPROVED** [review]

**Pull request merged**